### ISSUE TITLE
Template format error: Outputs count 72 is greater than max allowed 60

### ISSUE THREAD

**Issue by str3tch** [serverless/serverless#3135]
# This is a Bug Report 

## Description

* What went wrong?

Our Serverless project now has over 30 endpoints. Serverless cannot deploy, because it configures CloudFormation to output the Logical ID and the ARN's of the created endpoints, and there is a maximum limit of 60 entries in the Outputs section in CloudFormation. So more than 30 endpoints and you'll hit this issue.

* What did you expect should have happened?

In it's current format, I guess I expect this to happen. What I would like to happen, is that we can set a flag to *not* output the Id's and ARN's after stack creation. Then we can optionally overcome this limit on Outputs. 

(I want to raise a PR to fix this, but the guidelines say I should raise an issue first).

* What was the config you used?
* What stacktrace or error message from your provider did you see?

```
Serverless Error ---------------------------------------

 Template format error: Outputs count 72 is greater than
 max allowed 60
```

For feature proposals:
* What is the use case that should be solved.

For us, we do not need the ARN's of the created lambda functions. And I'm sure there are other people too who do not need this. So this feature should be optional (with default to ON)

* If there is additional config how would it look

I would think something like this

```
sls deploy --noOutputs
```

Similar or dependent issues:
* https://github.com/serverless/serverless/issues/2387

This issue above is related, but is more to do with the size of the template, which we have no issue with. 

## Additional Data

* ***Serverless Framework Version you're using***:
1.5
* ***Operating System***:
Windows 10 / Ubuntu

**Comment by str3tch** [comment]
p.s. this is our current dirty hack. The idea is to make this code optional depending if the command line option has been specified.

https://github.com/integratedresearch/serverless/commit/49b3cf840bd110ec1e13a3068a5c49e63f15ff04


(also, we mentioned this problem on the forums http://forum.serverless.com/t/severless-error-output-greater-than-60/877 )

**Comment by rowanu** [comment]
This will get better in 1.6 - see @pmuens's final comment on #2853.

**Comment by str3tch** [comment]
thanks @rowanu - it does look like those PR's will help fix it. I can't make out from the comments whether I can completley disable the outputting of function ARNs.. we don't need them at all, so I'd like to turn them off. 

We can hang on with our dirty hack until 1.6 is released then :)

**Comment by pmuens** [comment]
Hey @str3tch thanks for reporting and @rowanu thanks for pointing to the resolving issue.

We'll release v1.6 on January 31st (see: https://github.com/serverless/serverless/milestone/21) which will include the fix 👍 

I'll close this issue for now as it should be resolved once v1.6 is released.

**Comment by pmuens** [comment]
BTW. There's also this plugin which will remove all the CloudFormation outputs: https://github.com/Brigad/serverless-remove-cf-outputs-plugin

Of course this is only useful when you don't use `Outputs` at all...
